<?xml version='1.0' encoding='UTF-8'?>
<root version="2.3.8">
 <tabbed_widget name="Main Window" parent="main_window">
  <Tab tab_name="Energy and spring pos" containers="1">
   <Container>
    <DockSplitter count="2" sizes="0.5;0.5" orientation="-">
     <DockArea name="...">
      <plot style="Lines" mode="TimeSeries">
       <range left="0.000060" bottom="-0.018954" top="0.776611" right="0.066320"/>
       <limitY/>
       <curve name="/pendulum_pos/w" color="#1f77b4"/>
      </plot>
     </DockArea>
     <DockSplitter count="2" sizes="0.5;0.5" orientation="|">
      <DockArea name="...">
       <plot style="Lines" mode="TimeSeries">
        <range left="0.000000" bottom="-0.079114" top="0.758120" right="0.066320"/>
        <limitY/>
        <curve name="/joint_states/arm_spring/position" color="#1ac938"/>
        <curve name="/pendulum_pos/z" color="#9467bd"/>
       </plot>
      </DockArea>
      <DockArea name="...">
       <plot style="Lines" mode="TimeSeries">
        <range left="0.000000" bottom="-1.159708" top="1.783378" right="0.066320"/>
        <limitY/>
        <curve name="/joint_states/arm_spring/velocity" color="#17becf"/>
       </plot>
      </DockArea>
     </DockSplitter>
    </DockSplitter>
   </Container>
  </Tab>
  <currentTabIndex index="0"/>
 </tabbed_widget>
 <use_relative_time_offset enabled="1"/>
 <!-- - - - - - - - - - - - - - - -->
 <!-- - - - - - - - - - - - - - - -->
 <Plugins>
  <plugin ID="DataLoad CSV">
   <default time_axis=""/>
  </plugin>
  <plugin ID="DataLoad ROS bags">
   <use_header_stamp value="false"/>
   <use_renaming_rules value="true"/>
   <discard_large_arrays value="true"/>
   <max_array_size value="100"/>
  </plugin>
  <plugin ID="DataLoad ULog"/>
  <plugin ID="MQTT Subscriber"/>
  <plugin ID="ROS Topic Subscriber">
   <use_header_stamp value="false"/>
   <use_renaming_rules value="true"/>
   <discard_large_arrays value="true"/>
   <max_array_size value="100"/>
  </plugin>
  <plugin ID="UDP Server"/>
  <plugin ID="WebSocket Server"/>
  <plugin ID="ZMQ Subscriber"/>
  <plugin ID="ROS /rosout Visualization" status="idle"/>
  <plugin ID="ROS Topic Re-Publisher" status="idle"/>
 </Plugins>
 <!-- - - - - - - - - - - - - - - -->
 <previouslyLoaded_Datafiles/>
 <previouslyLoaded_Streamer name="ROS Topic Subscriber"/>
 <!-- - - - - - - - - - - - - - - -->
 <customMathEquations/>
 <snippets/>
 <!-- - - - - - - - - - - - - - - -->
</root>

